modules/concepts/services/_index.md

```diff
@@ -117,6 +117,18 @@ If you need more details about dependency injection and how services work in the
 their documentation about the [Service Container](https://symfony.com/doc/4.4/service_container.html).
 {{% /notice %}}
 
+##### Exclude index.php files when adding wildcard resource
+
+When adding resources to your module namespace, with a wildcard, exclude all `index.php` files (used for security, redirecting to FO). 
```

Comment:
```suggestion
When adding resources with a wildcard to your module namespace, make sure to exclude all `index.php` files (this file is used for security; it is redirecting to FO to prevent reading from the directory). If you don't do it, you might be redirected whenever you open your shop.
```